packages/epo-react-lib/src/content-blocks/SimpleTable/styles.ts

```diff
@@ -50,6 +50,7 @@ export const Title = styled.dt<SimpleTableColorProps>`
   ${cContentRte}
   ${accessibleLink}
 
+  margin: 0;
```

Comment:
found a browser styling thing here and fixed it, it was probably ok as-is with a reset in place